packages/react/src/exports/index.ts

```diff
@@ -138,6 +138,12 @@ export {
 
 export { useWallet } from '../hooks/useWallet.js'
 
+export {
+  useOpenOrders,
+  type UseOpenOrdersParameters,
+  type UseOpenOrdersReturnType,
+} from '../hooks/useOpenOrders.js'
+
```

Comment:
extremely small nit, can we alphabetize exports

Reply:
🫡